docs/src/api/class-browsertype.md

```diff
@@ -75,6 +75,7 @@ This methods attaches Playwright to an existing browser instance.
 * langs: js
 - `params` <[Object]>
   - `wsEndpoint` <[string]> A browser websocket endpoint to connect to.
+  - `extraHTTPHeaders` <[Object]<[string], [string]>> Additional HTTP headers to be sent with web socket connect request. Optional.
```

Comment:
Let's wait for Joel to remove `params` ?

Reply:
sure